plugins/introduction.rst

```diff
@@ -4,11 +4,18 @@ Introduction
 Install
 -------
 
-Install the plugin client in your project with Composer_:
+The plugin client and the core plugins are available in the `php-http/client-common`_ package:
 
 .. code-block:: bash
 
-    $ composer require php-http/plugins
+    $ composer require php-http/client-common
+
+.. _php-http/client-common: https://github.com/php-http/client-common
+
+.. versionadded:: 1.1
+    The plugins were moved to the clients-common package in version 1.1.
+    If you work with version 1.0, you need to require the separate `php-http/plugins` package.
```

Comment:
... package and the namespace is ``Http\Client\Plugin`` rather than ``Http\Client\Common``